extensions/linear/package.json

```diff
@@ -30,6 +30,37 @@
           "description": "When enabled, \"Created via Raycast\" signature will be added at the bottom of the description.",
           "required": false,
           "default": true
+        },
+        {
+          "name": "autofocusTitle",
+          "type": "checkbox",
+          "title": "Autofocus",
+          "label": "Automatically focus title field",
```

Comment:
could it be a dropdown to pick the field to focus instead?

Reply:
It could, but it feels a bit overkill to implement it since there are no user needs for that right now. What do you think?

Reply:
Even if there are only 2 options for now, I'd feel that it would be both clearer and more extensible (keep in mind that if we want more options later, a checkbox won't be enough and we will lose the preference

Reply:
Are you talking about a dropdown for all fields or only for the team and title fields?

If that's for all fields, I feel like picking the field to focus from a dropdown doesn't seem right as I don't see why a user would want to autofocus a field past the title. I get the need for the fields before the title but not after.

The implementation looks also a bit harder to implement since the `autofocus` prop would have to be conditionally applied to every field. Also, some fields are conditionally hidden if there are no data (e.g cycles), so a user could very well select the "Cycles" field in the preference even if their team has no cycles. That would mean that these edge cases would need to be handled as well.

Reply:
A dropdown to choose just between the team and the title. It would make it clearer what you are changing it from (which isn't clear with the checkbox)